uservice_ccutter/plugins/lsst_technote_bootstrap.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
+"""Field-calculating plugins for lsst-technote-bootstrap.  For each
+field you want substituted, make a function whose name the field name
+you want to change and which takes a dictionary as input.  It should
+returns the value that field should have after substitution.  Then
+call the plugins substitute() function with the cookiecutter template
+type as the first string argument and your
+dictionary-requiring-substitution as input, and it will change the
+values in that dictionary.
+
+"""
+import github3
+from apikit import BackendError
+from .generic import current_year
+
+ORGHASH = {"sqr": "lsst-sqre",
+           "dmtn": "lsst-dm",
+           "smtn": "lsst-sims",
+           "ldm": "lsst"}
+
+
+def serial_number(auth, inputdict):
```

Comment:
Do you have any Hungarian ancestry? Don't answer that -- HR probably won't let me ask.